test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
The review guidelines tell me that I shouldn't let you use MiniTest, but have you write your tests in RSpec instead.